src/networks/evm/constants.ts

```diff
@@ -163,6 +183,12 @@ export const EDITOR_VOTING_STRATEGIES = [
           type: 'integer',
           title: 'Decimals',
           examples: ['18']
+        },
+        symbol: {
+          type: 'string',
+          maxLength: 6,
+          title: 'Symbol',
+          examples: ['COMP']
```

Comment:
We should use `e.g.` on placeholders examples:
```suggestion
          examples: ['e.g. COMP']
```